langworld_db_data/adders/listed_value_adder.py

```diff
@@ -76,9 +76,9 @@ def _add_to_inventory_of_listed_values(
             )
         )
 
-        # Check if index_to_assign is -1 or belongs to range of indices in value_indices_to_inventory_line_numbers
+        # Check if passed index is valid
         last_index_in_feature = value_indices_to_inventory_line_numbers[-1]["index"]
-        acceptable_indices_to_assign = [-1] + [i for i in range(1, last_index_in_feature)]
+        acceptable_indices_to_assign = set([-1] + [i for i in range(1, last_index_in_feature)])
```

Comment:
```suggestion
        acceptable_indices_to_assign = {i for i in range(1, last_index_in_feature)}.union({-1})
```
not sure it's more readable, but it avoids unnecessary type conversions. 
Also, remember that the logic here is, as far as I can see, not entirely correct. I think that `last_index_in_feature` should also be allowed, and right now it is not because of the nature of ranges

Reply:
UPD nevermind, I'm ok with applying `set()` to a list for the sake of readability